router/core/graphql_prehandler.go

```diff
@@ -454,21 +463,26 @@ func (h *PreHandler) handleOperation(req *http.Request, variablesParser *astjson
 		isApq     bool
 	)
 
-	if operationKit.parsedOperation.IsPersistedOperation {
+	if operationKit.parsedOperation.IsPersistedOperation || h.operationBlocker.SafelistEnabled || h.operationBlocker.LogUnknownOperationsEnabled {
```

Comment:
This could also be a separate function to have clarity when exactly we enter this code branch.
E.g. you can explain what it means that the operation is persisted OR safelist is enabled or logUnknown is enabled. Why enter the code branch in these 3 cases?